src/components/Comment/Comment.tsx

```diff
@@ -56,8 +56,8 @@ const commentCss = css`
   display: block;
   clear: left;
   ${textSans.small()}
-  margin-top: 0.375rem;
-  margin-bottom: 0.5rem;
+  margin-top: ${space[2]}px;
+  margin-bottom: ${space[3]}px;
```

Comment:
[Best of both worlds](https://github.com/guardian/source/blob/master/src/core/foundations/src/space.ts#L19):

```suggestion
  margin-top: ${remSpace[2]}px;
  margin-bottom: ${remSpace[3]}px;
```

Reply:
nice! I want to know how you did a multi-line suggestion?

Reply:
It's not super-obvious:

1. Click and drag the line numbers to highlight multiple lines
2. Click the blue ➕ icon next to the bottom line number
3. Click "insert a suggestion"